k8s.io/configmap-nginx.yaml

```diff
@@ -190,6 +190,8 @@ data:
           rewrite ^/oncall$          https://storage.googleapis.com/kubernetes-jenkins/oncall.html;
           rewrite ^/partner-request$ https://docs.google.com/forms/d/e/1FAIpQLSdN1KtSKX2VAOPGABFlShkSd6CajQynoL4QCVtY0dj76MNDKg/viewform redirect;
           rewrite ^/start$           http://kubernetes.io/docs/getting-started-guides/ redirect;
+          rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html
```

Comment:
looks like ` redirect` is optional, since these all start with `https`.
The semicolon is not optional. :)